LEAF_Request_Portal/index.php

```diff
@@ -91,17 +91,23 @@ function customTemplate($tpl)
         {
             $categoryArray[$key] = array_map('XSSHelpers::xscrub', $cat );
         }
+
+        $servicesArray = $stack->getServices2();
```

Comment:
This should be `$form->getServices2()`. `FormStack` has no method `getServices2()` so this throws an error when creating new Requests.

Reply:
Fixed